src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeDeletionTest.java

```diff
@@ -36,6 +36,9 @@
 
     @Before
     public void setUp() throws Exception {
+        // Note: this test uses Azure blob store credentials instead of a Sas token
+        // as it needs to delete files in its unhappy path. Sas tokens have only the
+        // following permissions: create, write and list
```

Comment:
is 'create' and the same as 'write'?
Here's the code from the `SasTokenGeneratorService`:
```
policy.setPermissions(EnumSet.of(WRITE, LIST));
```